megazords/ios/MozillaAppServices.h

```diff
@@ -7,9 +7,10 @@
 FOUNDATION_EXPORT double MegazordClientVersionNumber;
 FOUNDATION_EXPORT const unsigned char MegazordClientVersionString[];
 
-#import "RustFxAFFI.h"
+#import "uniffi_fxa_client-Bridging-Header.h"
 #import "RustPasswordAPI.h"
 #import "RustLogFFI.h"
 #import "RustPlacesAPI.h"
 #import "RustViaductFFI.h"
 #import "GleanFfi.h"
+#import "uniffi_nimbus-Bridging-Header.h"
```

Comment:
Extremely petty nit: I think we should group all the uniffi ones together in this list :-)